algorithms/src/snark/marlin/ahp/indexer/indexer.rs

```diff
@@ -63,18 +63,18 @@ impl<F: PrimeField, MM: MarlinMode> AHPForR1CS<F, MM> {
 
             index_info,
         } = Self::index_helper(c)?;
-        let joint_arithmetization_time = start_timer!(|| "Arithmetizing A");
-
+        let id = CircuitId(Circuit::<F, MM>::hash(&index_info, &a, &b, &c).unwrap());
```

Comment:
Once we fix we change `hash` to output `CircuitId` directly, we don't need to explicitly create this wrapper.